config/gen.go

```diff
@@ -0,0 +1,3 @@
+//go:generate go run ./local/generate_toml.go --oracle-config-path ./local/oracle.toml
+
+package config
```

Comment:
we can use this file for staging and build out some CI process to update all of our configs when needed.